dist/aepp-sdk.js

```diff
@@ -0,0 +1,88 @@
+var _classCallCheck = require("@babel/runtime/helpers/classCallCheck");
```

Comment:
did you make sure to build this right before creating the PR? just want to double check because I would usually forget this :)

Reply:
I haven't.

Reply:
Done.